mcs/class/System/Test/System.Net.Sockets/SocketTest.cs

```diff
@@ -512,7 +512,7 @@ public void SocketErrorTest ()
 			sock.BeginConnect (ep, new AsyncCallback(SocketError_callback),
 				sock);
 
-			if (SocketError_event.WaitOne (2000, false) == false) {
+			if (SocketError_event.WaitOne (5000, false) == false) {
```

Comment:
Is it the only place where the timeout needs to be increased?

Reply:
This was just something I saw run local tests on Windows. Not related to this change, but since I hit it locally I though we could just increase it. I have not seen issues with this on CI.

Reply:
Just cosmetic; but we might want to extract those timeouts into constants to make it easier to change them.